libmachine/provision/configure_swarm.go

```diff
@@ -50,6 +50,9 @@ func configureSwarm(p Provisioner, swarmOptions swarm.Options, authOptions auth.
 			swarmOptions.Strategy,
 			advertiseMasterInfo,
 		)
+		if swarmOptions.IsExperimental {
```

Comment:
It's idiom to have double dash flags in these (i.e. `--experimental`), so can you update this PR to do that instead?